visualization/app/codeCharta/ui/metricChooser/metricChooser.component.scss

```diff
@@ -38,4 +38,13 @@ color-metric-chooser-component {
     float: right;
   }
 
+  .md-select-value .metric-max-value {
+    display: none;
+  }
+
+}
+
+.metric-max-value {
```

Comment:
you can change the color and size according to your concept here. Can be merged in my opinion. Looks good !

Reply:
@mrpatpat Yea I tried that, but when running my local server it doesn't adapt the style. Not in chrome, Firefox nor Safari. Did it work for you?

Reply:
Yes, it did. That is strange.

Reply:
As its not really a problem, I would suggest to merge it. Maybe we will find the solution later.

Reply:
btw: I just merged the master into this branch and now the grey color kinda shows up on my local server. Maybe... I forgot to pull your changes 😄